content/source/docs/extend/testing/acceptance-tests/index.html.md

```diff
@@ -56,7 +56,7 @@ signature:
 
 Terraform requires an environment variable `TF_ACC` be set in order to run
 acceptance tests. This is by design, and intended to prevent developers from
-incurring unintended charges when running tests. The easiest way to run
+incurring unintended changes when running tests. The easiest way to run
```

Comment:
This isn't a typo, but instead refers to accidentally running up an actual-cash-money bill with your cloud provider. How about this, though? 

```suggestion
incurring a larger-than-expected bill with a cloud provider when running tests. The easiest way to run
```